.drone.yml

```diff
@@ -25,47 +21,73 @@ steps:
       DOCKER_PASSWORD:
         from_secret: QUAY_ROBOT_TOKEN
       DOCKER_USERNAME: ukhomeofficedigital+hocs_quay_robot
+    depends_on:
+      - clone
 
----
-
-# we're doing this as a promote so one can manually run it
-# normally this is defined in hocs-data's .drone.yml
+  - name: build & push latest
+    image: plugins/docker
+    settings:
+      registry: quay.io
+      repo: quay.io/ukhomeofficedigital/hocs-toolbox
+      tags:
+        - latest
+    environment:
+      DOCKER_PASSWORD:
+        from_secret: QUAY_ROBOT_TOKEN
+      DOCKER_USERNAME: ukhomeofficedigital+hocs_quay_robot
+    when:
+      branch:
+        - main
+    depends_on:
+      - clone
 
+---
 kind: pipeline
 type: kubernetes
-name: teardown
+name: deployment
 
 trigger:
   event:
     - promote
+  target:
+    exclude:
+      - "*-prod"
+      - teardown
 
 steps:
-  - name: teardown gamma
+  - name: deploy to not-prod
```

Comment:
Consider automatically deploying latest to all useful environments. I don't think we need to version these properly.

Reply:
As discussed as the deployments will use latest and are scaled down by default to 0. It will pull the latest image whenever it starts up. For other instances, the user can just `rollout restart` the deployment to pull the new latest.